docs/api-docs/cart-and-checkout/custom-order-confirmation.md

```diff
@@ -0,0 +1,91 @@
+# Installing a Custom Order Confirmation Page
+
+<div class="otp" id="no-index">
+
+
+### On this page
+
+- [Generate the JavaScript loader file](#generate-the-javascript-loader-file)
+- [Hosting a custom order confirmation page](#hosting-a-custom-order-confirmation-page)
+- [Installing a custom order confirmation page](#installing-a-custom-order-confirmation-page)
+- [Related resources](#related-resources)
+
+</div>
+
+You can engage with your customers after purchase with a custom order confirmation page. This article outlines how to package a custom order confirmation file, and install a custom confirmation file via the control panel.
+The installation process is the same as [Installing a Custom Checkout](https://developer.bigcommerce.com/stencil-docs/customizing-checkout/installing-custom-checkouts), except you will modify a different [file](https://github.com/bigcommerce/checkout-js/blob/master/src/app/order/OrderConfirmation.tsx) in [Open Source Checkout](https://github.com/bigcommerce/checkout-js).
+
+## Generate the JavaScript loader file
+The first step to installing a custom order confirmation page is generating the loader file. If creating both a custom checkout and a custom order confirmation page using checkout-js, the generated loader file is the same. However, you do not need to create a custom checkout in order to have a custom order confirmation page. You can install a custom order confirmation page and still use the default checkout. The loader file must be responsible for the following:
+
+* Loading all the required assets, including CSS files
+* Running the app
+
+### Packaging custom order confirmation page with Open Source Checkout
+
+If using Open Source Checkout, you can generate the loader file by following these steps:
+
+1. Navigate and locate the Open Source Checkout folder using the terminal, and run `npm ci` to download dependencies
+2. Run `npm run build` to generate a `/dist/` folder with all the files
+
+When using Open Source Checkout to view changes and test implementations through the UI, deploy, and install the custom confirmation page. More details are below.
```

Comment:
Stating that there are "more details below" is too vague. Specify and link to the section that is most relevant.